### PR TITLE
Connections UI updates

### LINKED ISSUE #1039: Connections design iterations
To discuss with the team, we've had a few requests for Connection design iterations.

- Make the top-of-page connection element appear more substantial. Feedback was that it feels a little bit of an afterthought after a large header and not really a content element.
- When there's no image for a connection, don't show an image preview.
- Minimize white space at bottom of Pub when there's not much body content (e.g. https://rapidreviewscovid19.mitpress.mit.edu/pub/uxusa1t8/release/1)
- Create a blank connection type for internal linking that is more for display than graph purposes.
- Ability to customize carousel vs. list vs. description show/hide defaults
- Tooltip for authors with affiliation etc

### PR DESCRIPTION
**Pull request by 3mcd**
This PR contains the following changes (closes #1039):

- Removed placeholder thumbnail image (when external resource doesn’t have its own thumbnail).
- Add two new Pub-level options for displaying Pub Connections: default listing view (carousel/list), and an option to show the target resource's description by default.
- Better grouping of form controls in the Pub Connection editor.

_Test Plan_

- Create a Pub Connection and confirm both the dashboard views and editor view update correctly when toggling the new connection options.
- Ensure the UI tweaks match the designs in Figma.

### CODE REVIEW

File: client/components/DragDropListing/DragDropListing.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { Draggable, Droppable } from 'react-beautiful-dnd';
 
 require('./dragDropListing.scss');
 
-type OwnProps = {
+export type DragDropListingProps = {
```
Comment (idreyn) [inline client/components/DragDropListing/DragDropListing.tsx:10]: Ha, I just did this in my own branch

File: client/components/PubEdge/PubEdge.tsx [hunk 2]
```diff
@@ -75,25 +70,28 @@ const getValuesFromPubEdge = (pubEdge, communityData, viewingFromTarget) => {
 	return {};
 };
 
-type Props = OwnProps & typeof defaultProps;
+type Props = PubEdgeProps;
```
Comment (idreyn) [inline client/components/PubEdge/PubEdge.tsx:73]: Nit, but this is probably not necessary

**Review by idreyn — APPROVED** [review]
LGTM — there's something up with the dev DB right now that is preventing me from testing this, but the code looks solid!

File: client/components/PubEdge/PubEdge.tsx (hunk 2 above)
Comment (3mcd) [reply client/components/PubEdge/PubEdge.tsx:73]: Woops, totally missed this.

**Pull request merged**